docs/docs.md

```diff
@@ -0,0 +1,585 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }[]
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  redAlliance: string[]
+  blueAlliance: string[]
+  // UTC date - match predicted time
+  time: string
+  // example: qm3
+  key: string
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/reports/{team}`
+
+## `PUT`
+
+### Request
+
+```ts
+type Data = {
+  teleop: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+  auto: (
```

Comment:
I think this data needs to include auto mode name

Reply:
:heavy_check_mark: